src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Infrastructure/Repositories/ProviderRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SFA.DAS.Payments.DCFS.Infrastructure.Data;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Infrastructure.Data;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Infrastructure.Repositories
+{
+    public class ProviderRepository : DcfsRepository, IProviderRepository
+    {
+        private const string ProviderSource = "Reference.Providers";
+        private const string ProviderColumns = "UKPRN, "
+                                             + "IlrSubmissionDateTime";
+        private const string SelectProviders = "SELECT " + ProviderColumns + " FROM " + ProviderSource;
```

Comment:
Nooooo.. please don't use that way of building the string. And please put this into a shared project, `SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure` with integration tests.